src/CheckOut.js

```diff
@@ -729,6 +730,13 @@ class CheckOut extends React.Component {
           }
           label={<FormattedMessage id="ui-checkout.awaitingPickupLabel" />}
         />
+        <NotePopupModal
+          id="user-popup-note-modal"
+          domainName="users"
+          entityType="user"
+          popUpPropertyName="popUpOnCheckOut"
+          entityId={patron?.id}
```

Comment:
`entityId` proptype is string, if patron has not id we will return `undefined` so maybe it will be better
```suggestion
          entityId={patron?.id || ''}
```

Reply:
`NotePopupModal` has a default prop for `entityId`, so there's no need to pass it here